tests/kernel/boot_ltp.pm

```diff
@@ -45,6 +47,16 @@ sub run {
 
     assert_script_run('export LTPROOT=/opt/ltp; export LTP_COLORIZE_OUTPUT=n TMPDIR=/tmp PATH=$LTPROOT/testcases/bin:$PATH');
 
+    if (script_run('test -d $LTPROOT') != 0) {
```

Comment:
I think this should only be enabled for scenarios where you expect it to happen. We don't want someone to accidentally schedule 100 LTP installations by configuring an LTP test with the wrong image.

Reply:
OK, I'll put it only for baremetal tests (`LTP_BAREMETAL == 1`).

Reply:
@richiejp on the other hand we agreed with metan to switch to LTP installing from package (there are nightly builds, so it saves time just to install it).

Reply:
Yeah, the VM image grows quite a lot during LTP compilation/installation, so it might be best to remove that and install from the package on every test. That could fix the boot issue on Tumbleweed x86.

Reply:
@richiejp ok, I'll do 2 things: 1) install LTP from package (https://progress.opensuse.org/issues/53951, but keep possibility to install from git) and then implement this (install LTP on each test).